packages/@react-spectrum/list/stories/ListView.stories.tsx

```diff
@@ -350,3 +360,41 @@ function EmptyTest() {
     </div>
   );
 }
+
+function AsyncList() {
+  interface StarWarsChar {
+    name: string,
+    url: string
+  }
+
+  let list = useAsyncList<StarWarsChar>({
+    async load({signal, cursor}) {
+      if (cursor) {
+        cursor = cursor.replace(/^http:\/\//i, 'https://');
+      }
+
+      // Slow down load so progress circle can appear
+      await new Promise(resolve => setTimeout(resolve, 1500));
+      let res = await fetch(cursor || 'https://swapi.py4e.com/api/people/?search=', {signal});
+      let json = await res.json();
+      return {
+        items: json.results,
+        cursor: json.next
+      };
+    }
+  });
+  return (
+    <ListView
```

Comment:
I wouldn't mind a dropdown (really I want a knonb) to change the density between normal, compact, or spacius.

Reply:
For a follow up PR perhaps